### DISCUSSION TITLE
小爱音箱之前有执行文本命令和播放文本，转到官方这个功能被砍了

### DISCUSSION THREAD

**Discussion by qq173972819** [XiaoMi/ha_xiaomi_home#413] [Ideas]
需要输入文本执行命令功能，需要播放文本功能，在别的地方执行某些功能需要音箱播报提醒

**Comment by huhuime** [comment]
这几个都有，只是教程写的不明确
![image](https://github.com/user-attachments/assets/e23033a1-166b-43b3-9c98-331d1a83e35f)

**Comment by JimmyKmi** [comment]
参考 https://note.p.lzj.ac.cn/s/bd635048-ae38-43c9-9f31-d61175a01cf6/doc/tts-DeMFxdNX0s
